sct@skulptist.de/info.json

```diff
@@ -0,0 +1,3 @@
+{
+    "author": "Matthias Hermsdorf"
```

Comment:
This should be your GitHub username. The name of the applet probably should too but that is a loosely defined convention.

Reply:
I will change the author. Changing the applet name is possible. My page is https://skulptist.de and my own name is so long. If you insist, I would change it.

Reply:
I don't insist. 😅

It's fully up to you either way.